vectordb_bench/__init__.py

```diff
@@ -35,6 +35,8 @@ class config:
 
 
     K_DEFAULT = 100  # default return top k nearest neighbors during search
+    RESULTS_LOCAL_DIR = pathlib.Path(__file__).parent.joinpath("results")
```

Comment:
Already have a RESULTS_LOCAL_DIR in Line:29, please remove this

Reply:
good catch~